tests/draft4/additionalItems.json

```diff
@@ -40,6 +40,11 @@
         },
         "tests": [
             {
+                "description": "fewer items is valid",
```

Comment:
Maybe "fewer number of items present" and "equal number of items present" is good enough.

It's mostly the generality of the description below that seems out of place to me now, since it says "no additional items present" but that applies mostly equally to this test case.